docs/guide/installation.rst

```diff
@@ -0,0 +1,100 @@
+
+To install SunPy launch a system command prompt or the 'Anaconda Prompt' (under Windows).
+First configure conda for to add the `conda-forge channel <https://conda-forge.org/>__`::
+
+    conda config --add channels conda-forge
+    conda config --set channel_priority strict
+
+and now to install sunpy::
+
+    conda install sunpy
+
+This will install sunpy and every package it needs to function.
+
+Updating sunpy
+--------------
+
+You can update to the latest version by running::
+
+    conda update sunpy
+
+Installing sunpy on top of an existing scientific Python environment
+--------------------------------------------------------------------
+
+These instructions assume you have a scientific Python distribution with access to the ``pip`` command installed.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+We do provide compiled binaries for sunpy for Linux and Mac OS X (we do not compile our C extension on Windows), so you might not need a C compiler.
+If there is no compiled binary, you will need a C compiler (e.g., ``gcc`` or ``clang``) to be installed as we have a C library within sunpy that is built at install time.
+If you use Miniconda, you can get these compilers from there.
+On Linux, using the package manager for your distribution will usually be the easiest route, while on MacOS X you will need the XCode command line tools.
+
+Using ``pip``
+^^^^^^^^^^^^^
+
+.. note::
+    We strongly recommend using a `Python virtual environment <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/>`__ so any Python packages you install will be kept separate from your Operating System Python.
+    You can use Conda as a virtual environment manager instead of one of the more common Python virtual environment managers.
+
+sunpy consists of many sub-modules that each have their own requirements.
+You do not need to fulfil all the requirements if you only intend on using parts of sunpy.
+It is however it **strongly recommended** to have all the dependencies installed.
+This ensure that your version of sunpy is fully operational.
+
+To install sunpy with ``pip`` including all optional dependencies (**recommended**), simply run::
+
+    pip install sunpy[all]
+
+To install sunpy with no optional dependencies::
+
+    pip install sunpy
+
+To install sunpy with map- and timeseries-based dependencies::
+
+    pip install sunpy[map,timeseries]
+
+The available options are: ``[asdf]``, ``[dask]``,``[database]``,``[instr]``,``[image]``,``[jpeg2000]``,``[map]``,``[net]``,``[timeseries]``,``[visualization]``.
+
+If you want to develop sunpy we would strongly recommend reading the `Newcomers' Guide <https://docs.sunpy.org/en/latest/dev_guide/newcomers.html>`__.
+
+.. note::
+    If you get a ``PermissionError`` this means that you do not have the required administrative access to install new packages to your Python installation.
+    In this case you may consider using the ``--user`` option to install the package into your home directory.
```

Comment:
I would probably recommend venv here before `--user` saying something like "if you really don't want to use venv, you can always use `--user`".

Reply:
Fixed.